### PR TITLE
google app signing

### PR DESCRIPTION
**Pull request by MonicaG**
This source for this page is from [Google App Signing](https://github.com/bcgov/mobile-signing-service/wiki/Google-App-Signing) document. I removed references about the Developer Experience Team responsibilities. This guide is focused only on developers. Internal documentation to the Developer Experience Team will be handled separately

### CODE REVIEW

File: docs/google_app_signing.md [hunk 1]
```diff
@@ -1 +1,34 @@
-# google app signing
\ No newline at end of file
+# Google Signing
```
Comment (kaitlyn-rosenburg) [inline docs/google_app_signing.md:1]:
```suggestion
# Google app signing
```

File: docs/google_app_signing.md [hunk 2]
```diff
@@ -1 +1,34 @@
...
+
+
+To learn about code signing, please review Google's [Sign your app](https://developer.android.com/studio/publish/app-signing) documentation.
```
Comment (kaitlyn-rosenburg) [inline docs/google_app_signing.md:4]:
```suggestion
To learn about code signing, review Google's [Sign your app](https://developer.android.com/studio/publish/app-signing) documentation.
```

File: docs/google_app_signing.md [hunk 3]
```diff
@@ -1 +1,34 @@
...
+
+This guide describes how an app team should work with Google keys. You must use the Province of BC's Google Play developer account to distribute your app.
```
Comment (kaitlyn-rosenburg) [inline docs/google_app_signing.md:6]: move up to be intro sentence of page

File: docs/google_app_signing.md [hunk 4]
```diff
@@ -1 +1,34 @@
...
+
+React Native apps need extra setup to use the keys. Please see the [React Native Signing Guide](google_react_native_signing.md) for more details.
```
Comment (kaitlyn-rosenburg) [inline docs/google_app_signing.md:8]:
```suggestion
React Native apps need extra setup to use the keys. Review the [React Native Signing Guide](google_react_native_signing.md) for more details.
```

File: docs/google_app_signing.md [hunk 5]
```diff
@@ -1 +1,34 @@
...
+
+## Keys
+Google Play uses two keys:
+
+- upload key
```
Comment (kaitlyn-rosenburg) [inline docs/google_app_signing.md:13]:
```suggestion
- Upload key
```

File: docs/google_app_signing.md [hunk 6]
```diff
@@ -1 +1,34 @@
...
+- app signing key
```
Comment (kaitlyn-rosenburg) [inline docs/google_app_signing.md:14]:
```suggestion
- App signing key 
```

File: docs/google_app_signing.md [hunk 7]
```diff
@@ -1 +1,34 @@
...
+
+React Native apps need extra setup to use the keys. Please see the [React Native Signing Guide](google_react_native_signing.md) for more details.
+
+## Keys
+Google Play uses two keys:
+
+- upload key
+- app signing key 
+
+Please refer to Google's [Play App Signing](https://developer.android.com/studio/publish/app-signing#app-signing-google-play) documentation for an in-depth review of the two keys.
```
Comment (kaitlyn-rosenburg) [inline docs/google_app_signing.md:16]:
```suggestion
Use Google's [Play App Signing](https://developer.android.com/studio/publish/app-signing#app-signing-google-play) documentation for an in-depth review of the two keys.
```

File: docs/google_app_signing.md [hunk 8]
```diff
@@ -1 +1,34 @@
...
+
+### Upload Key
+
+The upload key signs your app for upload to the Google Play Store. The team's app release manager should [create the upload key](https://developer.android.com/studio/publish/app-signing#sign-apk). 
+
+If your app's code is in the [bcgov GitHub organization](https://github.com/bcgov), then please use a [GitHub Action](https://docs.github.com/en/actions) to build and sign your app with the upload key. Refer to Google's [Sign your app from command line](https://developer.android.com/build/building-cmdline#sign_cmdline) documentation for instructions on how to do this. Store the key ([encoded as base64](https://docs.github.com/en/actions/security-guides/encrypted-secrets#storing-base64-binary-blobs-as-secrets)), its alias and the password as a [secret](https://docs.github.com/en/actions/security-guides/encrypted-secrets) in your app's repo.
```
Comment (kaitlyn-rosenburg) [inline docs/google_app_signing.md:22]:
```suggestion
If your app's code is in the [bcgov GitHub organization](https://github.com/bcgov), use a [GitHub Action](https://docs.github.com/en/actions) to build and sign your app with the upload key. 

Refer to Google's [Sign your app from command line](https://developer.android.com/build/building-cmdline#sign_cmdline) documentation for instructions on how to do this. Store the key ([encoded as base64](https://docs.github.com/en/actions/security-guides/encrypted-secrets#storing-base64-binary-blobs-as-secrets)), its alias and the password as a [secret](https://docs.github.com/en/actions/security-guides/encrypted-secrets) in your app's repo. 
```

File: docs/google_app_signing.md [hunk 9]
```diff
@@ -1 +1,34 @@
...
+
+### Upload Key
+
+The upload key signs your app for upload to the Google Play Store. The team's app release manager should [create the upload key](https://developer.android.com/studio/publish/app-signing#sign-apk). 
+
+If your app's code is in the [bcgov GitHub organization](https://github.com/bcgov), then please use a [GitHub Action](https://docs.github.com/en/actions) to build and sign your app with the upload key. Refer to Google's [Sign your app from command line](https://developer.android.com/build/building-cmdline#sign_cmdline) documentation for instructions on how to do this. Store the key ([encoded as base64](https://docs.github.com/en/actions/security-guides/encrypted-secrets#storing-base64-binary-blobs-as-secrets)), its alias and the password as a [secret](https://docs.github.com/en/actions/security-guides/encrypted-secrets) in your app's repo. 
+
+
+See the [bc-wallet-mobile](https://github.com/bcgov/bc-wallet-mobile/blob/main/.github/workflows/main.yaml) project for an example of a project using GitHub Actions.
```
Comment (kaitlyn-rosenburg) [inline docs/google_app_signing.md:25]:
```suggestion
The [bc-wallet-mobile](https://github.com/bcgov/bc-wallet-mobile/blob/main/.github/workflows/main.yaml) project is a good example of a project using GitHub Actions.
```

File: docs/google_app_signing.md [hunk 10]
```diff
@@ -1 +1,34 @@
...
+
+If you are using a different git host, then please see its documentation on how to create a CI/CD pipeline. Use its secret feature to store the key and password securely.
```
Comment (kaitlyn-rosenburg) [inline docs/google_app_signing.md:27]:
```suggestion
If you're using a different git host, review its documentation on how to create a CI/CD pipeline. Use its secret feature to store the key and password securely.
```

File: docs/google_app_signing.md [hunk 11]
```diff
@@ -1 +1,34 @@
...
+
+If you do not use a CI/CD pipeline, then store the key and password somewhere securely.  However, we encourage teams to use a CI/CD pipeline. It provides repeatable builds and safely stores the credentials.
```
Comment (kaitlyn-rosenburg) [inline docs/google_app_signing.md:29]:
```suggestion
If you don't use a CI/CD pipeline, store the key and password somewhere securely. 
We encourage teams to use a CI/CD pipeline. It provides repeatable builds and safely stores the credentials.
```

File: docs/google_app_signing.md [hunk 12]
```diff
@@ -1 +1,34 @@
...
+
+### App Signing Key
+
+Please use the [Play App Signing](https://developer.android.com/studio/publish/app-signing#app-signing-google-play) service to sign your app. If you do not use this service and you lose your app’s signing key, you lose the ability to update your app. Google's [Using Play App Signing](https://developer.android.com/studio/publish/app-signing#enroll) documentation describes how to setup it up for your app.
```
Comment (kaitlyn-rosenburg) [inline docs/google_app_signing.md:33]:
```suggestion
Use the [Play App Signing](https://developer.android.com/studio/publish/app-signing#app-signing-google-play) service to sign your app. If you don't use this service and you lose your app’s signing key, you lose the ability to update your app. Google's [Using Play App Signing](https://developer.android.com/studio/publish/app-signing#enroll) documentation describes how to setup it up for your app.
```

**Review by kaitlyn-rosenburg — CHANGES_REQUESTED** [review]

File: docs/google_app_signing.md (hunk 2 above)
Comment (MonicaG) [reply docs/google_app_signing.md:4]:
```suggestion
This guide describes how an app team should work with Google keys. You must use the Province of BC's Google Play developer account to distribute your app.

To learn about code signing, review Google's [Sign your app](https://developer.android.com/studio/publish/app-signing) documentation.
```

File: docs/google_app_signing.md (hunk 3 above)
Comment (MonicaG) [reply docs/google_app_signing.md:6]:
```suggestion

```

**Comment by MonicaG** [comment]
@kaitlyn-rosenburg - I've applied the suggestions you requested. You can [review them here](https://github.com/bcgov/mobile-developer-guide/pull/14/commits/46c28bef7cc16137b11f6230ee266f9ff4e51d28?diff=split&w=0#diff-d824c3986b12eee9403daa846c2f02a2bf8be78cfb520b4da0d229721ad955e2).

**Review by kaitlyn-rosenburg — APPROVED** [review]

**Pull request merged**